guide/using-matlab.md

```diff
@@ -93,10 +92,10 @@ In order to create a cross-platform controller, it is recommended to use the
 We recommend to use the latest MATLAB version on an up-to-date operating system.
 
 Note that 64-bit versions of Webots are not compatible with 32-bit versions of
-MATLAB. Webots comes only in 64-bit flavour and therefore it can only
+MATLAB. Webots comes only in 64-bit flavours and therefore it can only
```

Comment:
This comment is not related with your change, but could you please change it to the American English version ``flavors``?